src/components/ExpensiTextInput/BaseExpensiTextInput.js

```diff
@@ -118,51 +119,57 @@ class BaseExpensiTextInput extends Component {
             ignoreLabelTranslateX,
             innerRef,
             autoFocus,
+            errorText,
```

Comment:
Can we define `propTypes` and `defaultProps` for these?

Reply:
Thanks, done!